CustomApps/lyrics-plus/style.css

```diff
@@ -89,7 +90,7 @@
 
 .lyrics-lyricsContainer-LyricsUnsyncedPadding {
 	display: flex;
-	height: 25vh;
+	height: 10vh;
```

Comment:
Doesn't look quite right when the track is instrumental, should leave it as-is or find another solution.
![image](https://github.com/spicetify/spicetify-cli/assets/77577746/9837e921-bb29-4870-a24e-125b6f84db6a)
![image](https://github.com/spicetify/spicetify-cli/assets/77577746/4ff7f271-a8e1-4239-a4ef-8787a19fe23a)

Reply:
Doesn't look quite right when the track is instrumenta.
![image](https://github.com/spicetify/spicetify-cli/assets/77577746/9837e921-bb29-4870-a24e-125b6f84db6a)

```suggestion
	height: calc(50vh - 12px - 90px - var(--lyrics-font-size));
}

.lyrics-lyricsContainer-UnsyncedLyricsPage:has(.lyrics-lyricsContainer-LyricsLine:nth-child(3)) .lyrics-lyricsContainer-LyricsUnsyncedPadding {
	height: 10vh;
}
```